### PR TITLE
chore: update tests to work with iOS 13.0 (w/ Azure)

### PR DESCRIPTION
**Pull request by imurchie**
Just #1071 with the Azure config from #1008, to check.

### CODE REVIEW

**Comment by imurchie** [comment]
Web tests are SSL tests covered by #1070

File: test/functional/basic/element-e2e-specs.js [hunk 1]
```diff
@@ -108,6 +109,9 @@ describe('XCUITestDriver - element(s)', function () {
 
   describe('contentSize', function () {
     it('should get the contentSize of a table', async function () {
+      if (util.compareVersions(UICATALOG_CAPS.platformVersion, '>=', '13.0')) {
+        return this.skip();
```
Comment (mykola-mokhnach) [inline test/functional/basic/element-e2e-specs.js]: why the test is skipped?

File: test/functional/basic/gesture-e2e-specs.js [hunk 2]
```diff
@@ -246,7 +261,9 @@ describe('XCUITestDriver - gestures', function () {
         } catch (e) {
           // Otherwise, pull down the middle of the top of the Simulator
           x = width / 2;
-          y0 = height - 5;
+          y0 = UICATALOG_CAPS.deviceName.toLowerCase().includes('iphone x')
```
Comment (mykola-mokhnach) [inline test/functional/basic/gesture-e2e-specs.js]: I don't like tests, which constantly require hardcoded value updates. How about removing them completely?

File: test/functional/desired.js [hunk 3]
```diff
@@ -2,24 +2,29 @@ import _ from 'lodash';
 import path from 'path';
 import glob from 'glob';
 import fs from 'fs';
-import { system } from 'appium-support';
+import { system, util } from 'appium-support';
 
 
+// translate integer environment variable to a boolean 0=false, !0=true
+function checkFeatureInEnv (envArg) {
+  let feature = parseInt(process.env[envArg], 10);
```
Comment (mykola-mokhnach) [inline test/functional/desired.js]: how about

```
return `${process.env[envArg]}`.trim() === '0'
```

**Pull request closed without merging**